domains/zoro.xen.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "xenmods",
+        "email": "ilumomin04@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "db112aae-90b3-4d1b-a302-e4da8d2cae38.id.repl.co"
```

Comment:
I can't seem to reach this website.

Reply:
it is up now.

Reply:
@xenmods What ezactly does this website do?